src/Sylius/Bundle/UserBundle/Form/Type/GenderType.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
+ */
+class GenderType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $resolver->setDefaults(array(
+            'choices' => array(
+                0 => '',
+                1 => 'sylius.gender.male',
+                2 => 'sylius.gender.female',
```

Comment:
hmmm any reason for storing gender as an integer?  It seems a bit unneccessary to obfuscate what is either 'm' or 'f' or an empty string here. We just store gender as a char(1) since that's all you'll ever need, I don't think there are any other genders that we need to consider... :)

We actually have a GenderBundle for this (Which was a bit overkill I must admit, but we have product genders as well which is slightly different and we use them for the distinction).

So we end up with:

```
parameters:
    reiss.gender.form.type.gender.class: Reiss\GenderBundle\Form\Type\GenderChoiceType
    reiss.genders.people:
        W: reiss.gender.people.women
        M: reiss.gender.people.men
    reiss.genders.products:
        W: reiss.gender.product.women
        M: reiss.gender.product.men
        U: reiss.gender.product.unisex
```

And then in our gender form:

```
class GenderChoiceType extends AbstractType
{
    /**
     * Associative gender array
     *
     * @var array
     */
    protected $genders = array();

    /**
     * @param array $genders
     */
    public function __construct(array $genders, $name)
    {
        $this->genders = $genders;
        $this->name = $name;
    }

    /**
     * {@inheritdoc}
     */
    public function setDefaultOptions(OptionsResolverInterface $resolver)
    {
        $resolver->setDefaults(array(
            'choices' => $this->genders
        ));
    }

```